akka-http/src/main/scala/akka/http/javadsl/server/HttpService.scala

```diff
@@ -7,7 +7,6 @@ package akka.http.javadsl.server
 import akka.actor.ActorSystem
 import akka.http.scaladsl.{ server, Http }
 import akka.http.scaladsl.Http.ServerBinding
-import akka.http.scaladsl.server.RouteResult
```

Comment:
might be better to just `import akka.http.scaladsl` to make usage really obvious